src/Core/src/Handlers/Button/ButtonHandler.iOS.cs

```diff
@@ -151,6 +157,27 @@ static void SetControlPropertiesFromProxy(UIButton platformView)
 			}
 		}
 
+		internal static void UpdateAttributedTitle(IButtonHandler handler, ITextStyle textStyle)
```

Comment:
Could we move this as an extension method in `ButtonExtensions`?

Reply:
Sure, I just did it. I have one question though. This method needs `fontManager` which currently is getting resolved by `IButtonHandler`. Should I keep this line of code: `var fontManager = handler.GetRequiredService<IFontManager>();`  inside this new extension method or pass `IFontManager` as a parameter to the `UpdateAttributedTitle` method and call `handler.GetRequiredService<IFontManager>();` before calling `UpdateAttributedTitle`?

Reply:
Good question. We usually pass it as parameter. For example, here https://github.com/dotnet/maui/blob/85adc13a249fe4ac59da6518ea5dc70055ee569c/src/Core/src/Handlers/Button/ButtonHandler.iOS.cs#L125C4-L125C65

Reply:
Changed in the last commit